font/src/ft/mod.rs

```diff
@@ -37,6 +38,7 @@ struct Face {
     render_mode: freetype::RenderMode,
     lcd_filter: c_uint,
     non_scalable: Option<FixedSize>,
+    has_color: bool,
```

Comment:
We should acknowledge the renderer that it should check `buf` part and scale colored bitmaps to fit into cells. I guess it should help on other platforms and fix an issue with rendering emojis with the right color.

Reply:
It turned out that this has_color field could be pretty useful, so I guess we should keep it.